solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -501,6 +505,24 @@ public static Throwable getWrappedException(Throwable e) {
     return e;
   }
 
+  /**
+   * When using a InputStreamResponseParser, the raw output is available in the response under the
+   * key STREAM_KEY.
+   */
+  public static String getOutputFromInputStreamResponseParserResponse(NamedList<Object> response)
```

Comment:
As the functionality you add is specific to InputStreamResponseParser, I think a static method there makes more sense.

Also, I want to avoid adding more helper methods to STCJ4.  Tons of methods on this class... lots of legacy stuff abounds.

Reply:
Love it!